roles/nodepool/files/etc/nodepool/elements/nodepool/cleanup.d/90-validate-dib-image

```diff
@@ -0,0 +1,57 @@
+#!/bin/bash
+
+function reporter () {
+  message="$1"
+  shift
+  echo
+  echo "${message}"
+  for (( i=0; i<${message}; i++ )); do
+    echo -n '-'
+  done
```

Comment:
@mlangbehn This is a function I've used in bash scripts for years. It interprets the length of the message and prints the correct number of dashes. So if your message was "Installing python", it would look like this:

```
Installing python
----------------------
```